Assets/MixedRealityToolkit/Inspectors/Utilities/HandednessInspectorGUI.cs

```diff
@@ -0,0 +1,29 @@
+﻿
+using UnityEditor;
+using UnityEngine;
+
+public static class HandednessInspectorGUI
+{
+    private static readonly GUIContent[] HandednessSelections =
+    {
+        new GUIContent("Left Hand"),
+        new GUIContent("Right Hand"),
+        new GUIContent("Both Hands"),
+    };
+
+    public static void DrawControllerHandednessDropdown(SerializedProperty controllerHandedness)
```

Comment:
Please comment public methods, [refer to this guideline](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/DocumentationGuide.html#property-method-event-summary-blocks)

Reply:
It would be good to mention that this actually modifies the input controllerHandedness as well.